frontend/packages/shared/src/utils/layoutSetsUtils.ts

```diff
@@ -15,6 +15,8 @@ export const getLayoutSetIdValidationErrorKey = (
   if (oldLayoutSetId === newLayoutSetId) return null;
   if (!validateLayoutNameAndLayoutSetName(newLayoutSetId)) return 'ux_editor.pages_error_format';
   if (!newLayoutSetId) return 'validation_errors.required';
+  if (newLayoutSetId.length === 1)
```

Comment:
Great work by adding a validation to verify that requirements are met. But I think we also should add one more validation that verifies that the input is not just two empty whitespace. :)

```typescript
if(newLayoutSetId.trim() === ''`) {
    return 'validation_errors.required';
 }
 ```

Reply:
Do we need this actually ? because we have already these messages:
- `Feltet er påkreved` when the input is empty.
- `Navnet må bestå av bokstaver (a-z), tall, eller "-", "_" og ".".` when we have one, two or more white spaces.